test/servo.js

```diff
@@ -180,6 +180,27 @@ exports["Servo"] = {
     test.done();
   },
 
+  degreeChange: function(test) {
+    test.expect(2);
+
+    this.servo = new Servo({
+      pin: 11,
+      board: board
+    });
+
+    this.servo.to(0);
+
+    this.servo.to(180, 1000, 100);
+    this.servo.to(180, 1000, 100);
+
+    this.clock.tick(1010);
```

Comment:
This test won't catch the error. Both Servo.to() calls fire on the same tick when the distance is still 180. The test needs to wait for the first one to complete before calling the second, or calling servo.to(180) and then servo.to(180, 1000)